src/app/util/af-types.h

```diff
@@ -191,6 +191,16 @@ enum
   EMBER_AF_ENDPOINT_ENABLED  = 0x01,
 };
 
+#ifdef DOXYGEN_SHOULD_SKIP_THIS
```

Comment:
The DOXYGEN_SHOULD_SKIP_THIS bits are some sort of leftovers from the original silabs code that we never got around to cleaning up.  Just define a normal enum class here, please.  Put it in the `chip::app` namespace, probably, though we could also do `chip::app::DataModel`. No need to use all-shouty naming, and so on.

Basically: don't make it look like C code just because everything else in this file started out as C code.